src/main/java/org/opensearch/ad/MemoryTracker.java

```diff
@@ -50,7 +51,7 @@
 
     public enum Origin {
         SINGLE_ENTITY_DETECTOR,
-        MULTI_ENTITY_DETECTOR,
+        HC_DETECTOR,
```

Comment:
Seems this caused build failure. Make sure all places changed.

Reply:
sure, will make sure that in the last PR with all of the code.